web/src/app/core/services/authentication.service.ts

```diff
@@ -21,6 +21,7 @@ import { ProfileModel, LoginAuditModel } from '../../shared/models/index.model';
 })
 export class AuthenticationService {
 
+    private spinnerSubject: Subject<Boolean> = new Subject();
```

Comment:
This would be better in its own service, so the `AuthenticationService` is not included in many places where the `AuthenticationService` is not really required

Reply:
Created spinner service for the function and removed from authentication service. Thanks for suggestion